tritonbench/operators/fp8_gemm_rowwise/operator.py

```diff
@@ -198,6 +199,17 @@ def _triton(self, xq, wq, x_scale, w_scale, bias) -> Callable:
             use_warp_specialization=self.warp_specialization,
         )
 
+    @register_benchmark(enabled=HAS_TRITON)
```

Comment:
Are there any restrictions on where this can run e.g. H100 vs MI300? If not this looks good.